### ISSUE TITLE
Improve disabled button state

### ISSUE THREAD

**Issue by donjo** [18F/identity-style-guide#49]
Our current disabled buttons are not meeting accessibility contrast requirements and disabled buttons may or may [not be very accessible at all](https://axesslab.com/disabled-buttons-suck/). Moving forward we should either consider increasing our contrast or electing to not include them in the system.

**Comment by adunkman** [comment]
When this is addressed, we’ll want to remove the entries to `hideElements` from d703641e049e2f747578a46d3a168ce7969ca184 which ignore checkbox and radio button labels — once the contrast is acceptable, they should no longer need to be ignored.

**Comment by nickttng** [comment]
Took a look at USWDS 2.0 beta, and their [disabled state](https://v2.designsystem.digital.gov/components/buttons/) has more contrast than their 1.0 version. I'll ask in #uswds-public Slack if their contrast update was intentionally to solve that accessibility issue.

**Comment by nickttng** [comment]
Update from #uswds-public Slack:

**Maya B:**
> Hi Nick, if the 2.0 buttons have more contrast it's likely from conforming to the new color palettes. WCAG provides an exception for disabled state color contrast requirements, though there is some reasoning to not use disabled buttons at all: https://axesslab.com/disabled-buttons-suck/

**Dan W:**
> yeah, there's all kinds of potential pitfalls with the disabled button. we may still get to the point of actively discouraging it. while we don't need to worry about contrast for its display, there are other, perhaps stronger reasons to consider it bad practice. And yeah, just as `@maya` said, there are some color changes from 1.x -> 2.0 that are largely an effect of moving to our broader, more rigorous color palette

Sounds like we could go in one of those directions:
1. **Increase the contrast -** Possibly taking from USWDS 2.0.0 Beta's disabled `#c9c9c9` (see the example in screenshot below; Also, [viewable in Figma](https://www.figma.com/file/W5xHB8EdrCghG2IWW74tI53b/Issue-Disabled-Button-State?node-id=0%3A1)). A short-term "solution" to move a bit forward, although it doesn't really solve anything.
2. **Elect not include a disabled button state -** We can actively discourage this practice of relying solely on visual contrast as 'good-enough' and/or educate the team not to use the light-gray disabled button and instead, adopt a better practice questioning why Login/we would use disabled button in the first place. However, the risk is that later, newer devs/engineers may implement their own disabled gray states because the design system doesn't explicitly make a rule on it.

Any thoughts or other solutions that we can explore?

![login-issue-disabled-button-state](https://user-images.githubusercontent.com/6327082/53989126-e4ffb900-40ea-11e9-9dc7-ff9598a0d2db.jpg)

**Comment by nickttng** [comment]
@donjo Just noticed that that new disabled button has #767676, so for large text, it passes AAA while for normal text, it passes AA. I think it's okay to close this issue?
